packages/thirdweb/src/extensions/prebuilts/deploy-published.ts

```diff
@@ -181,22 +184,67 @@ export async function deployContractfromDeployMetadata(
       const initializeTransaction = await getInitializeTransaction({
         client,
         chain,
-        deployMetadata: deployMetadata,
+        deployMetadata,
         implementationContract,
         initializeParams,
         account,
         modules,
       });
 
-      return deployViaAutoFactory({
+      // TODO: remove this once the modified version of TWCloneFactory
+      // has been published under the thirdweb wallet
+      const modifiedCloneFactoryContract = getContract({
+        client,
+        address: "0xB83db4b940e4796aA1f53DBFC824B9B1865835D5", // only deployed on OP and zora testnets
+        chain,
+      });
+
+      return await deployViaAutoFactory({
         client,
         chain,
         account,
-        cloneFactoryContract,
+        cloneFactoryContract: modifiedCloneFactoryContract,
```

Comment:
Override it only for crosschain deployments for now.

Reply:
addressed